numpy-stubs/__init__.pyi

```diff
@@ -792,3 +797,131 @@ class AxisError(ValueError, IndexError):
     def __init__(
         self, axis: int, ndim: Optional[int] = ..., msg_prefix: Optional[str] = ...
     ) -> None: ...
+
+# Functions from np.core.fromnumeric
+_Mode = Literal["raise", "wrap", "clip"]
+_Order = Literal["C", "F", "A"]
+_PartitionKind = Literal["introselect"]
+_SortKind = Literal["quicksort", "mergesort", "heapsort", "stable"]
+
+# Various annotations for scalars
+_ScalarGeneric = TypeVar("_ScalarGeneric", bound=generic)
+_ScalarBuiltin = Union[str, bytes, dt.date, dt.timedelta, bool, int, float, complex]
+_Scalar = Union[_ScalarBuiltin, generic]
+
+# An array-like object consisting of integers
+# TODO: If possible, figure out a better way to deal with nested sequences
+_Int = Union[int, integer]
+_ArrayLikeInt = Union[
```

Comment:
Continuation of https://github.com/numpy/numpy-stubs/pull/67#discussion_r412327691.
Not the most elegant solution, but it sort of does the job.